docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java

```diff
@@ -13,7 +13,7 @@
 import org.apache.pekko.stream.Materializer;
 
 @SuppressWarnings("deprecation")
-public class AkkaHttp1020MigrationExample {
+public class PekkoHttp1020MigrationExample {
```

Comment:
Just accidentally stumbled over this, but seems like this one would correctly be named Akka, as there is no Pekko 10.2 Migration

Reply:
I vote for keeping the new name - it easier to explain all the names now have Pekko in them - as opposed to explaining on a case by case basis. One extra option is to just to remove this test.

Reply:
I don't have any firm opinion, but unfirmly: removing it seems like the most right option then.

Reply:
Its likely these will be removed as they don't make sense in Pekko, but trademarks (and other legal matters) generally takes precedence over other matters.